run_unittests.py

```diff
@@ -3236,6 +3236,21 @@ def test_introspect_projectinfo_subprojects(self):
         }
         self.assertDictEqual(res, expected)
 
+    def test_introspection_target_subproject(self):
+        testdir = os.path.join(self.common_test_dir, '46 subproject')
+        self.init(testdir)
+        res = self.introspect('--targets')
+
+        expected = {
+            'sublib': "sublib",
```

Comment:
Please keep the quotes consistent with the rest of the function (`'` instead of `"`)
```suggestion
            'sublib': 'sublib',
```

Reply:
fixed